views/extras.tmpl

```diff
@@ -188,14 +186,11 @@
     var ws; // websocket global
     async function createWebSocket(loc) {
         // wait a bit to prevent websocket churn from drive by page loads
-        console.log('Waiting to create websocket');
         $('#connection').removeClass('hidden');
         var uri = getSocketURI(loc)
         ws = new MessageSocket(uri);
         await sleep(3000);
```

Comment:
You asked why we have this.  The reason is there are many bouncing visitors who load the page and do not stay long.  I think crawlers do this often.  There is very high websocket connection churn without a short delay.